app/assets/javascripts/likes.js

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
I can see that when you run this, you're having an issue with the "like count" of all images being incremented whenever you click on a "like" link.